reactDashboard/src/components/DataCharts/DataChartsUtils.js

```diff
@@ -110,8 +121,7 @@ export function commonChartConfiguration(datasets) {
         ],
       },
       legend: {
-        position: "bottom",
-        onClick: (e) => e.stopPropagation(),
+        display: false,
```

Comment:
While needing a legend is debatable if there is a single trace and its meaning is clear from the chart title (I'd have one anyway), a legend is necessary if there is more than one trace (as in the regional insights chart)

Reply:
Good point - will look into this also

Reply:
![image](https://user-images.githubusercontent.com/22414785/95985805-f6f42280-0e1c-11eb-9d44-c48cd7284a0f.png)

Reply:
back in for both regional and overview

Reply:
![image](https://user-images.githubusercontent.com/22414785/95986269-9fa28200-0e1d-11eb-87f1-51f7a3acc10e.png)

Reply:
adjusted label box width